src/app/cmdoptions.cpp

```diff
@@ -372,7 +372,7 @@ QStringList QBtCommandLineParameters::paramList() const
     // torrent paths or URLs.
 
     if (!savePath.isEmpty())
-        result.append(QLatin1String("@savePath=") + savePath);
+        result.append(QLatin1String("@savePath=") + savePath.toString());
```

Comment:
One more time... There is no point in converting it to a native format in order to transfer it to another qBittorrent instance, where it will be converted back to uniform format inside Path.